sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -33,27 +34,6 @@ import org.apache.spark.sql.catalyst.util.DateTimeUtils
 
 /**
  * :: Experimental ::
- * Information about updates made to stateful operators in a [[StreamingQuery]] during a trigger.
```

Comment:
Thats actually the opposite of how most code in SQL is laid out, so I think it would be better to avoid this change.  The logic here is declarations that are use later should come first (references before declaration make it harder to read), and stuff at the end of the file is kind of hidden.